### DISCUSSION TITLE
upload file and all its references (if changed)

### DISCUSSION THREAD

**Discussion by chk1xn** [Natizyskunk/vscode-sftp#438] [Q&A] [answered]
Hi,

I'm completely new to Visual Studio Code so sorry if this is a silly Q. I'm aware I might simply be missing something obvious, so please bear with me...

I've installed SFTP to be able to upload files to the various web spaces I'm using from various ISPs.
Coming from Dreamweaver I got used to a handy option that would ask if I'd like to upload any dependent files along with the one I'm about to transfer. This included all files types that are referenced within that file, eg any css, js, pdf, images etc.

I've scanned thru the documentation of VSC and SFTP but haven't found an answer yet. Is this something that can be accomplished with the "sync" option or the "file watcher"? but I have a feeling "sync" only works with Github? and the "file watcher" only checks if local files have been modified outside of VSC?

Thanks for any hints/tips/pointers...

**Comment by Natizyskunk** [comment, marked as answer]
@chk1xn,
This option is not available for now but i'm keeping it in my mind. I'll definitely take a look at it when I've more time.

**Reply by chk1xn** [reply]
thanks very much for not forgetting me and giving me hope things will get better. :-)

**Reply by Natizyskunk** [reply]
Aha you're welcome, saddly for the last two years I haven't been able to work on this project as much as I wanted for personnal reasons but I'll try my best to at least update it with really necessary features.
